src/ui/pages/AssetInfo/AssetDescription.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useRef, useState } from 'react';
+import type { AssetFullInfo } from 'src/modules/zerion-api/requests/asset-get-fungible-full-info';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import { TextAnchor } from 'src/ui/ui-kit/TextAnchor';
+import { UIText } from 'src/ui/ui-kit/UIText';
+import { VStack } from 'src/ui/ui-kit/VStack';
+import LinkIcon from 'jsx:src/ui/assets/new-window.svg';
+import { UnstyledButton } from 'src/ui/ui-kit/UnstyledButton';
+import * as helperStyles from 'src/ui/style/helpers.module.css';
+
+/**
+ * Took this component from Web App
+ * https://github.com/zeriontech/pulse-frontend/blob/master/src/z/dumb/TextPreview/index.tsx
+ */
+const TextPreview = ({ text }: { text: string }) => {
+  const ref = useRef<HTMLDivElement | null>(null);
+  const [isClamped, setIsClamped] = useState<false | true | undefined>(
+    undefined
+  );
+  // on Safari line-camp works correctly only with inline elements inside
+  const paragraphs = text.split(/(\r?\n|\r)+/g);
+  const markup: React.ReactNode[] = [];
+  paragraphs.forEach((block, index, array) => {
+    markup.push(<span key={index}>{block}</span>);
+    markup.push(<br key={array.length + index} />);
+  });
+
+  useEffect(() => {
+    if (ref.current) {
```

Comment:
Looks like it could be possible to just:
`if (ref.current && ref.current.scrollHeight > ref.current.clientHeight) { ... `